app.js

```diff
@@ -24,24 +24,27 @@ function User (username, password, library, tagLibrary) {
-  var newUserEl = document.getElementById('newUser');
-  newUserEl.addEventListener('submit', function(e) {newUser(e);},false);
-  var existingButton = document.getElementById('existingButton');
-  existingButton.addEventListener('click', function(e) {returnUserForm(e);},false);
-}
+  $('#loginForm').append('<form name="loginform" class="whiteText" id="newUser"><fieldset><legend>New User</legend><label>Username</label><input class="labelColor" type="text" name="usr" placeholder="username" required="required"><label>Password</label><input class="labelColor" type="password" name="pword" placeholder="password" required="required"><p id="msg"></p><input class="button-primary" type="submit" value="Create New User"></fieldset></form><input class="button-primary" type="submit" value="Switch to Login Page" id="existingButton">');
+    var newUserEl = document.getElementById('newUser');
+    newUserEl.addEventListener('submit', function(e) {newUser(e);},false);
+    var existingButton = document.getElementById('existingButton');
+    existingButton.addEventListener('click', function(e) {returnUserForm(e);},false);
+  }
+
+
 function returnUserForm (event) {
-  document.getElementById('loginForm').innerHTML = '';
-  document.getElementById('loginForm').innerHTML = '<form name="loginform" class="whiteText" id="returnUser"><fieldset><legend>Returning User</legend><label>Username</label><input class="labelColor" type="text" name="usr" placeholder="username" required="required"><label>Password</label><input class="labelColor" type="password" name="pword" placeholder="password" required="required"><p id="msg"></p><input class="button-primary" type="submit" value="Login"></fieldset></form><input class="button-primary" type="submit" value="Create New User" id="newButton">';
+  // document.getElementById('loginForm').innerHTML = '';
+  $('#loginForm').append('<form name="loginform" class="whiteText" id="returnUser"><fieldset><legend>Returning User</legend><label>Username</label><input class="labelColor" type="text" name="usr" placeholder="username" required="required"><label>Password</label><input class="labelColor" type="password" name="pword" placeholder="password" required="required"><p id="msg"></p><input class="button-primary" type="submit" value="Login"></fieldset></form><input class="button-primary" type="submit" value="Create New User" id="newButton">');
   var returnUserEl = document.getElementById('returnUser');
```

Comment:
What if we re-factored this blob of text to look nicer by entering new elements onto new lines?